modules/ebi-metagenomics/eggnogmapper/main.nf

```diff
@@ -9,38 +9,40 @@ process EGGNOGMAPPER {
 
     input:
     tuple val(meta), path(fasta)
-    path(eggnog_db)
+    tuple val(meta), path(annotation_hit_table)
     path(eggnog_data_dir)
+    path(eggnog_db)
     path(eggnog_diamond_db)
 
     output:
-    tuple val(meta), path("*.emapper.hits"), emit: csv
-    path "versions.yml",                     emit: versions
+    tuple val(meta), path("*.emapper.hits"), emit: hits, optional: true
+    tuple val(meta), path("*annotations*") , emit: annotations, optional: true
+    tuple val(meta), path("*orthologs*")   , emit: orthologs, optional: true
```

Comment:
Make sure we need the * after annotations and orthologs (and not in the .emapper.hits). I don't remember the actual produced file extensions